debezium-operator-api/src/main/java/io/debezium/operator/api/model/source/storage/schema/JdbcSchemaHistoryTableConfig.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.operator.api.model.source.storage.schema;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+
+import io.debezium.operator.api.config.ConfigMappable;
+import io.debezium.operator.api.config.ConfigMapping;
+import io.debezium.operator.api.model.DebeziumServer;
+import io.debezium.operator.docs.annotations.Documented;
+import io.sundr.builder.annotations.Buildable;
+
+@Documented
+@Buildable(editableEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder", lazyCollectionInitEnabled = false)
+public class JdbcSchemaHistoryTableConfig implements ConfigMappable<DebeziumServer> {
+
+    @JsonPropertyDescription("The name of the offset table")
+    @JsonProperty(required = false)
+    private String name;
+
+    @JsonPropertyDescription("DDL statement to create the schema history table")
+    @JsonProperty(required = false)
+    private String ddl;
+
+    @JsonPropertyDescription("Statement used to select from the schema history table")
+    @JsonProperty(required = false)
+    private String select;
+
+    @JsonPropertyDescription("Statement used to insert into the schema history table")
+    @JsonProperty(required = false)
+    private String insert;
+
+    @JsonPropertyDescription("Statement used to check existence of the schema history table")
+    @JsonProperty(required = false)
+    private String dataExistsSelect;
```

Comment:
Maybe this can be rename to `tableExistsSelect`?

Reply:
well, I took the name form the code. But yea, IIRC that's what it does. Will rename

Reply:
@mfvitale  ok, so no. The query is actually used to check whether there is at least one row in the table. Thus current name seems more fitting. WDYT?

Reply:
Ok then the description should be changed to reflect this.